csharp/src/Ice/Internal/MetricsAdminI.cs

```diff
@@ -880,7 +880,7 @@ override public IceMX.MetricsFailures[] getMapMetricsFailures(string viewName, s
             {
                 return view.getFailures(mapName, id);
             }
-            return new IceMX.MetricsFailures("", null); // null == dictionary
```

Comment:
This was bogus too. We actually verify in the generated constructor the second parameter is non-null. This code appears untested.